src/Root.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { useEffect } from 'react';
+import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage';
+
+export const Root = () => {
+  useEffect(() => {
+    document.documentElement.classList.add('has-navbar-fixed-top');
+  }, []);
```

Comment:
Why do you need this?

Reply:
it is from task people-table basics. so pages will look as original